tests/data_models.py

```diff
@@ -1,23 +1,22 @@
 from datetime import datetime
-from typing import Sequence
 import json
+from typing import Sequence
 
 from sqlalchemy import select, text
 
-from models.models import (
-    Permission,
-    Post,
-    User,
+from models import (
+    Filter,
     Message,
-    Thread,
-    Report,
     Notification,
-    Filter,
-    Role,
     NotificationSub,
+    Post,
+    Report,
+    Role,
+    User,
 )
 from models.db import SendADatabase
-
+from models.schemas.messages import Thread
```

Comment:
Is there a reason these two aren't imported from `models`?

Reply:
Cos vscode is crap at auto refactoring? 🤣

I'll fix